lib/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "settings": {
+    "import/resolver": {
+      "typescript": {}
+    }
+  }
+}
```

Comment:
The empty configuration is to override the default typescript conf ?

Reply:
Nope, this is a trick to enable eslint-import-resolver-typescript and indicate to eslint to use `tsconfig.json` next to the eslintrc file.

I think I could also write :
```json
{
  "settings": {
    "import/resolver": {
      "typescript": {
        "projet": "./"
      }
    }
  }
}
```